### ISSUE TITLE
Test fail for apple arm64 backend

### ISSUE THREAD

**Issue by zhkl0228** [merryhime/dynarmic#734]
In file tests/A32/test_arm_instructions.cpp, add test:

```
TEST_CASE("arm: svc", "[arm][A32]") {
    ArmTestEnv test_env;
    A32::Jit jit{GetUserConfig(&test_env)};
    test_env.code_mem = {
        0xef0001ee,  // svc #0x1ee
        0xe30a0071,  // mov r0, #41073
        0xeafffffe,  // b +#0
    };

    jit.SetCpsr(0x000001d0);  // User-mode

    test_env.ticks_left = 3;
    jit.Run();

    REQUIRE(jit.Regs()[0] == 41073);
    REQUIRE(jit.Regs()[15] == 0x00000008);
}

**Comment by merryhime** [comment]
Thanks for the report, fixed in 11d73fedde0c9eb22bfabf8994a7900a65238044.

**Comment by SachinVin** [comment]
Finally works with citra, thanks ❤
